pkgs/applications/graphics/litemdview/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, stdenv
+, fetchFromGitea
+, fetchpatch
+, gtkmm3
+, autoconf
+, automake
+, libtool
+, cairo
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "litemdview";
+  # litemdview -v
+  version = "0.0.32";
+
+  src = fetchFromGitea {
+    domain = "notabug.org";
+    owner = "g0tsu";
+    repo = "litemdview";
+    rev = "eacbbc837e";
+    hash = "sha256-sD0y+/7TogywrvftxEXT1EyBf3BXH9QWbawFVOF8xRc=";
+  };
+
+  buildInputs = [
+    gtkmm3
+  ];
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    pkg-config
+    libtool
+    cairo
```

Comment:
I don't know `libtool` and `cairo` well enough, though are you sure that these are `nativeBuildInputs`?

Which roughly means, they provide binaries that are required to run during compilation, rather than libraries that need to be linked and available during runtime?

This distinction is mostly useful for cross compiling, still should be done correctly.

If you can explain why they should be `nativeBuildInputs` I'm fine with that…

PS: Crosscompiling for `aarch64` worked though, and I was also able to run via `qemu`, therefore I am really not sure if this is correct or not. This probably has to be answered by someone who understands the crosscompiling better than me.

Reply:
:warning:  Not sure about this one

Reply:
As `cairo` felt very weird being here, I tried removing it completely from the inputs. The build suceeds even without, are you sure that `cairo` is a required dependency?

Reply:
```suggestion
```

Reply:
Strange, it didn't build for me without cairo
Maybe it wasn't cairo that was the problem, will try without it